src/renderer/videoJS.css

```
@@ -2136,7 +2136,7 @@ video::-webkit-media-text-track-display {
 .video-js .vjs-vtt-thumbnail-display {
     position: absolute;
     transition: transform .1s, opacity .2s;
-    bottom: 20px;
+    bottom:56px;
```

I don't think there is a reason to eliminate the space
```suggestion
    /* `bottom` was 20px, to avoid obstruction by time tooltip updated to current value */
    bottom: 56px;
```